### DISCUSSION TITLE
Panasonic GH6 generated files unplayable after merge

### DISCUSSION THREAD

**Discussion by bullishtq** [mifi/lossless-cut#1349] [Support]
merging 4k 422 10 bit files that are AVC in an MOV wrapper with 4 channel LPCM audio from Panasonic GH6 creates a file that is unplayable by any software, even VLC. VLC says it can't play 'aivx' file and Adobe Premiere doesn't accept the file to begin with. 

Having searched previous issues you mentioned in issue 280, you listed to add -strict -2 -vtag avc1 in terminal by copying the text from help menu and adding this line to the end but doing that just tells me 'ffmpeg not found'. Maybe I was supposed to do it somewhere in your software but I guess I don't know it that well...

However I was able to use the exact same line in Shutter Encoder which then created a much smaller, compressed file with only 1 audio channel instead of the original 4; which obviously is not acceptable. 

Hoping you could offer a solution/tutorial to sort this out and maintain the original codec/audio channels. much appreciated! Thanks.

**Comment by bullishtq** [comment]
EDIT: Restarted Computer and everything seems to be working great. All fixed by the looks of it.

**Reply by mifi** [reply]
Probably same problem as #280
